src/test/java/org/kohsuke/github/GHContentIntegrationTest.java

```diff
@@ -85,6 +89,17 @@ public void testGetDirectoryContentTrailingSlash() throws Exception {
         assertThat(entries.get(0).getUrl(), endsWith("?ref=main"));
     }
 
+    GHCommit getGHCommit(GHContentUpdateResponse resp)
+            throws GHException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
+        for (Method method : resp.getClass().getMethods()) {
+            if (method.getName() == "getCommit" && method.getReturnType() == GHCommit.class) {
+                return (GHCommit) method.invoke(resp);
+            }
```

Comment:
To get coverage on `GHCommit.GHAuthor` you'll need to call `GHAuthor commit.getAuthor()` using reflection as well I think.